astroplan/constraints.py

```diff
@@ -906,6 +990,60 @@ def is_observable(constraints, observer, targets, times=None,
     return np.any(constraint_arr, axis=1)
 
 
+def is_event_observable(constraints, observer, target, times=None,
```

Comment:
I really like how straightforward this is both as an API and in implementation, given the previous work (no action required, just "this is cool 😉").